CONTRIBUTING.md

```diff
@@ -165,6 +165,20 @@ make run-ui-dev
 Once you have the server up and running, you will be able to access the Meshery UI at `http://localhost:3000`. One thing to note is that for the UI dev server to work, you need Meshery server running on the default port of `9081`.
 Any UI changes made now will automatically be recompiled and served in the browser.
 
+### Running Meshery from IDE
+If you want to run Meshery from IDE like Goland, VSCode. set below environment variable
+```
+SAAS_BASE_URL=https://meshery.layer5.io
+PORT=9081
+DEBUG=true
+ADAPTER_URLS=mesherylocal.layer5.io:10000 mesherylocal.layer5.io:10001 mesherylocal.layer5.io:10002 mesherylocal.layer5.io:10003 mesherylocal.layer5.io:10004
```

Comment:
Looking good. Do we need to add a note about updating the user's `/etc/hosts` file in order for `mesherylocal.layer5.io` to resolve to `localhost`?

Reply:
updated the PR.